31 create bus endpoint

Marcus-Aastum opened the pull request:
Create an endpoint that returns bus routes for Tiller VGS

Closes #31

Marcus-Aastum committed:
Create bus endpoint

Create an endpoint that returns bus routes for Tiller VGS. Also bugfix for init_tables.sql

Add Bus-endpoint on backend #31

JorgenMU commented on Backend/info-skjerm-api/Model/BusDepartureJsonResponse.cs:
```diff
@@ -0,0 +1,17 @@
+    public struct BusRoute
+    {
+        public int busLine { get; set; }
+        public string name { get; set; }
```
could be better to say "destination" insted of buss "name"

Marcus-Aastum committed:
Change name to destination RE comment

Change name in busdeparturejsonresponse to destination as requested by @JorgenMU

Add Bus-endpoint on backend #31
